### PR TITLE
Settings (Added existing settings to config)

### PR DESCRIPTION
**Pull request by Vonsen**

### CODE REVIEW

File: engine_config.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
     "WindowSize": [
```
Comment (norda97) [inline engine_config.json:2]: Should this really be float?

File: engine_config.json [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
-        1920,
-        1080
+        1920.0,
```
Comment (norda97) [inline engine_config.json]: Why is this not an integer?

File: LambdaEngine/Source/Game/GameConsole.cpp [hunk 3]
```diff
@@ -220,6 +221,127 @@ namespace LambdaEngine
...
+		// Set Boolean in config
+		{
+			ConsoleCommand cmdSetBool;
+			cmdSetBool.Init("set_bool", true);
+			cmdSetBool.AddArg(Arg::EType::STRING);
+			cmdSetBool.AddArg(Arg::EType::BOOL);
+			cmdSetBool.AddDescription("Change boolean value in config file with a given key\n\t 'set_bool key bool'");
+			BindCommand(cmdSetBool, [this](GameConsole::CallbackInput& input)->void
+				{
+					if (!EngineConfig::SetBoolProperty(input.Arguments.GetFront().Value.String, input.Arguments.GetFront().Value.Boolean))
```
Comment (jonathan2222) [inline LambdaEngine/Source/Game/GameConsole.cpp]: Fix the commands. Use argument 0 for the string and argument 1 for bool.
The problem is the same for the other settings commands which change variables.

**Review by jonathan2222 — CHANGES_REQUESTED** [review]

File: LambdaEngine/Include/Engine/EngineConfig.h [hunk 4]
```diff
@@ -20,14 +20,14 @@ namespace LambdaEngine
         static int GetIntProperty(const String& propertyName);
         static double GetDoubleProperty(const String& propertyName);
```
Comment (norda97) [inline LambdaEngine/Include/Engine/EngineConfig.h]: Add tab alignment to getters

**Review by norda97 — APPROVED** [review]

**Review by jonathan2222 — APPROVED** [review]

**Pull request merged**